app/controllers/concerns/recaptcha_concern.rb

```diff
@@ -1,10 +1,6 @@
 module RecaptchaConcern
   extend ActiveSupport::Concern
 
-  included do
-    before_action :allow_csp_recaptcha_src, if: :recaptcha_enabled?
-  end
```

Comment:
❤️ I am a big fan of removing `included` blocks that wire up stuff like this and hide them from the controllers. Now that this is gone we could probably remove the `extend ActiveSupport::Concern` bit since this is a plain mixin now, right?

Reply:
I believe we can, yes! Updated in 848e9fbab.